src/main/scala/io/citrine/lolo/trees/regression/RegressionTree.scala

```diff
@@ -92,24 +92,18 @@ class RegressionTreeTrainingResult(
     rootTrainingNode: TrainingNode[Double],
     encoders: Seq[Option[CategoricalEncoder[Any]]]
 ) extends TrainingResult[Double] {
-  lazy val model = new RegressionTree(rootTrainingNode.modelNode, encoders)
-  lazy val importance: mutable.ArraySeq[Double] = rootTrainingNode.featureImportance
-  private lazy val importanceNormalized = {
-    if (Math.abs(importance.sum) > 0) {
-      importance.map(_ / importance.sum)
-    } else {
-      importance.map(_ => 1.0 / importance.size)
-    }
-  }
 
-  override def getModel(): RegressionTree = model
+  override lazy val model = new RegressionTree(rootTrainingNode.modelNode, encoders)
```

Comment:
`RegressionTree` and `RegressionTreeTrainingResult` seem like more candidates for turning into case classes

Reply:
(also `ClassificationTree`)